features/admin.actions.v1/components/pre-update-password-action-config-form.tsx

```diff
@@ -207,7 +241,8 @@ const PreUpdatePasswordActionConfigForm: FunctionComponent<PreUpdatePasswordActi
                 passwordSharing: {
                     ...(PEMValue ? { certificate: PEMValue } : {}),
                     format: values.passwordSharing
-                }
+                },
+                rule: payloadRule !== null ? payloadRule : undefined
```

Comment:
```suggestion
                rule: payloadRule ?? undefined
```

Ref: https://github.com/wso2/identity-apps/blob/master/docs/write-code/CODE_QUALITY.md#use-modern-js-features-to-keep-the-code-concise-and-readable

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7486/commits/34a926448e3cea96642355bfe473951501c89739